migrations/915-rename-addon-tags.sql

```diff
@@ -0,0 +1 @@
+RENAME TABLE `users_tags_addons` TO `addons_tags`;
```

Comment:
Table renames don't require a temporary table.

Also, the constraints are on the table itself, not on the related tables so we should be ok there. There are no constraints on either `tags` or `addons` pointing to `addons_tags`.